### DISCUSSION TITLE
empty latest_receipt_info

### DISCUSSION THREAD

**Discussion by felipechiodini** [imdhemy/laravel-in-app-purchases#156] [Q&A] [answered]
somes receipt-datas that I submit to apple API return the field latest_receipt_info empty, anyone know tell me why? 

![image](https://user-images.githubusercontent.com/50850744/195931419-15a3b41c-363e-4d4b-9618-31cd1a1f551c.png)

**Comment by imdhemy** [comment, marked as answer]
The `latest_receipt_info` only returns for receipts that contain auto-renewable subscriptions. This excludes transactions for consumable products that your app marks as finished.

**Reply by felipechiodini** [reply]
But I never selled others products that not are auto-renewable subscription before in my app.

**Reply by imdhemy** [reply]
@felipechiodini 

Could you please share a snippet of your code? 

It may be related to this issue. 

https://github.com/imdhemy/appstore-iap/issues/39

**Reply by felipechiodini** [reply]
Is only this, I just submit the receipt data that my app production is using.
![image](https://user-images.githubusercontent.com/50850744/196414161-baf55229-2b43-42e8-8c62-757be2d6cd1a.png)

**Reply by imdhemy** [reply]
Looks like it's not related to the mentioned issue.

Let's debug the raw response from the App store.

```php
$rawData = $receipt->toArray();
```

If the `latest_receipt_info` is not available, you may contact App store support.

**Reply by imdhemy** [reply]
Hey @felipechiodini any updates on this? what are your findings till now?

**Reply by felipechiodini** [reply]
Hi, I'm sorry don't awnser you before, but I still receiving latest_receipt_info empty, then I simple ignore it because I guess it is not an active subscription, I really think its about an old subscriptions that is not more currently active. 

Thanks for you support.

**Reply by imdhemy** [reply]
No need to apologize. Thank you for letting me know.
